Add `Astro.callAction()` guide (#10005)
withastro/docs: src/content/docs/en/guides/actions.mdx

}
```
:::

## Call actions from Astro components and server endpoints

> You may want to call actions directly from Astro component scripts or server endpoints. This is common when reusing logic from your actions in other server code.
> 
> Call actions using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](/en/guides/endpoints/#server-endpoints-api-routes)), passing the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:

Reviewer: ```suggestion
You can call actions directly from Astro component scripts using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](/en/guides/endpoints/#server-endpoints-api-routes)). This is common to reuse logic from your actions in other server code. 

Pass the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:
```
Just removing the whole "you might want to" in favour of just saying how to do it.

> Call actions using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](http://localhost:4321/en/guides/endpoints/#server-endpoints-api-routes)), passing the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:

Reviewer 2: It's quite optimistic to assume that every user always has a development server of the documentation up and running. ;)

```suggestion
Call actions using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](/en/guides/endpoints/#server-endpoints-api-routes)), passing the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:
```
